### ISSUE TITLE
Fixed length - cannot change document type

### ISSUE THREAD

**Issue by ijowerspricht** [umco/umbraco-nested-content#147]
After setting fixed length of nested content - ie. min and max is same. It automatically creates content of the first document type defined. And there is no way to remove one and add a different document type.

**Comment by leekelleher** [comment]
Hi @ijowerspricht - by setting the min and max to 1, that enables the single item mode - which is intentionally auto-created for you.  If you want to be able to select the doctype, then you'll need to set the min to 0 and the max to 1.

**Comment by ijowerspricht** [comment]
Yes, but what if I want to have exactly 12 items with the possibility of changing between doctypes? It seems like this is unpossible right now.

**Comment by leekelleher** [comment]
Hi @ijowerspricht - that use-case isn't currently possible. If you want a min & max of 12 items, then NestedContent would create those 12 items for you. If you don't want NC to auto-create them, then you'll need to set the min to a lower value.   I appreciate that may not be ideal for your use-case, but that's how it currently works.

Also to note, that NC (package version, _for this GitHub repo_) has been migrated to the Umbraco core, (in v7.7) so any issues would need to be raised on the [Umbraco issue tracker](http://issues.umbraco.org/issues/U4).
